src/user/models/auth/auth.service.ts

```diff
@@ -61,19 +62,19 @@ export class AuthService {
     return { accessToken: this.generateToken(user) };
   }
 
-  getSignMessage(address: string): string {
-    if (isEthereumAddress(address) || this.cryptoService.isBitcoinAddress(address))
+  getSignMessage(address: string, blockchain: Blockchain): string {
```

Comment:
Wouldn't it be better to remove the blockchain param and determine the blockchain in this method?